index.html

```diff
@@ -0,0 +1,156 @@
+              </ul>
+            </li>
+            <li>
+              <button class="header-btn" type="submit">Get a ticket</button>
+            </li>
+          </ul>
+        </nav>
+      </header>
+      <main class="event-main">
+        <section class="event-brif">
+          <h2 class="event-title-main">Egypt: The Time of Pharaohs</h2>
+          <h1 class="event-title-second">
+            Exhibition 2022 &mdash; London: Ancient Egypt
+          </h1>
+          <p class="event-description">
+            London Exhibition 2022 will take visitors into the distant past,
+            exploring the myths and realities of ancient Egyptian culture.
+          </p>
+          <p class="event-date">2022.10.10(Mon) ~ 23(Sun)</p>
+
+          <p class="event-location">
+            @ Saatchi Gallery, Duke of York's HQ, King's Rd, London SW3 4RY
+          </p>
+        </section>
+        <section class="main-program">
+          <h3 class="title">Main Program</h3>
+          <hr class="big-title-bar" />
+          <ul class="main-program-list">
+            <li class="main-program-item">
+              <i class="ph-line-segments main-icon"></i>
+              <h4>Sections</h4>
+              <p>
+                Have a walk thru our different Exhibition sections and
+                experience the Ancient Egypt vibe.
+              </p>
+            </li>
+            <li class="main-program-item">
+              <i class="ph-chalkboard-teacher main-icon"></i>
+              <h4>Lecture</h4>
+              <p>
+                Listen to the speakers, experienced archaeologists revelling in
+                the great mysteries of Ancient Egypt.
+              </p>
+            </li>
+            <li class="main-program-item">
+              <i class="ph-aperture main-icon"></i>
+              <h4>Photography</h4>
+              <p>
+                Famous photographers share their work with photos pyramids,
+                temples, the Nile etc.
+              </p>
+            </li>
+            <li class="main-program-item">
+              <i class="ph-chats main-icon"></i>
+              <h4>Forum</h4>
+              <p>
+                Join the Forum to connect with other scholars passionate about
+                this ancient civilization.
+              </p>
+            </li>
+            <li class="main-program-item">
+              <i class="ph-globe main-icon"></i>
+              <h4>Next</h4>
+              <p>
+                Find out about future exhibitions in London and across the
+                globe.
+              </p>
+            </li>
+          </ul>
+          <button type="submit" class="main-btn">Get a ticket</button>
+          <a class="see-program" href="#">See the whole program</a>
+        </section>
+        <section class="exhib-sections">
+          <h3 class="title">Exhibition Sections</h3>
+          <hr class="big-title-bar" />
+          <ul class="exhib-section-list"></ul>
+        </section>
```

Comment:
- [ ] The contents within the items of your main section are **not vertically aligned**, as can be seen in the original design. Kindly improve upon this aspect by ensuring that the contents within the items are vertically centered by ensuring that there is even space at both the top and bottom. 

**Your main program section:**
![image](https://user-images.githubusercontent.com/12745474/137213030-84d8ef22-dfa7-42e4-b209-11b76922a3cd.png)


**The original main program section:**
![image](https://user-images.githubusercontent.com/12745474/137212800-719180d3-5ba9-4e72-b7e7-6ea3454a10e1.png)